github/ci/prow/files/labels.yaml

```diff
@@ -414,3 +414,17 @@ repos:
         target: prs
         prowPlugin: trigger
         addedBy: prow
+  kubevirt/kubectl-virt-plugin:
```

Comment:
Is there a reason why we always copy the configuration per repo instead of putting them into default?

Reply:
That's because not all repos are using tide.

Reply:
So if we put them into the default section the tide plugin borks? Or what else breaks in that case?

Reply:
Nothing breaks. The repos would only get labels added which they don't use.